test/avio.jl

```diff
@@ -147,13 +148,23 @@ end
             while !eof(v)
                 read!(v, img)
             end
+
+            # Iterator interface
+            VT = typeof(v)
+            @test Base.IteratorSize(VT) === Base.SizeUnknown()
+            @test Base.IteratorEltype(VT) === Base.EltypeUnknown()
+
+            VideoIO.seekstart(v)
+            @test length(collect(v)) == VideoIO.TestVideos.videofiles[name].numframes
+            @test length(collect(v)) == 0
```

Comment:
```suggestion
            imgstack = collect(v)
            @test length(imgstack) == VideoIO.TestVideos.videofiles[name].numframes
            imgstack = nothing
            @test length(collect(v)) == 0
            GC.gc()
```

Reply:
Yeah good points... if CI is memory limited I probably should be using `collect` at all. I'll do some other test of the iterator interface that doesn't require storing all of the frames in memory at the same time.